### PR TITLE
Feature/implement send button

### LINKED ISSUE #15: Implement the logic for the send button
When the send button is clicked a request should be made to the commendation endpoint. use a FORM to complete this.

https://nextjs.org/docs/guides/building-forms#form-submission-without-javascript

### PR DESCRIPTION
**Pull request by ashlyndevries**
Closes #15, progress on #14.

### CODE REVIEW

**Comment by SpBills** [comment]
i love it

File: pages/api/commendation/index.ts [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
+      if (!req.body.reciever || !req.body.msg) {
+        // Sends a HTTP bad request error code
+        return res.status(400).json({ data: 'Recipient or message not found' })
+      }
+
+      const msg = req.body.msg as string;
+      const reciever = req.body.recipient as string;
+      const commendation = await createCommendation(msg);
+
+      res.status(200).json({ data: `${reciever} ${msg}` })
```
Comment (Konolo) [inline pages/api/commendation/index.ts:23]: the if statement and res.status(200)... are not needed. also don't split message here, do it in the createCommendation function.

File: pages/api/commendation/index.ts [hunk 2]
```diff
@@ -1,16 +1,26 @@
...
       res.json(commendation)
```
Comment (Konolo) [inline pages/api/commendation/index.ts:24]: this res.json(commendation) should be res.redirect("/")

File: lib/api/commendations/crud.ts [hunk 3]
```diff
@@ -2,10 +2,11 @@ import { Commendation, PrismaClient } from "@prisma/client";
 
 const prisma = new PrismaClient();
 
-export const createCommendation = async (message: string) => {
+export const createCommendation = async (msg: string) => {
   return await prisma.commendation.create({
     data: {
-      message
+      id: "",
+      message: msg,
```
Comment (Konolo) [inline lib/api/commendations/crud.ts:9]: the four fields that you need to include in data are {message, memberId, senderEmail, and recipientEmail}.

**Review by Konolo — CHANGES_REQUESTED** [review]
there are a few changes i would like you to make. run the command yarn prisma db pull. you also need to run yarn prisma generate.
https://docs.google.com/document/d/1VodIkAramYn71FtrvY1bfd6sa-KR_mP3fsTZSaKrcG4/edit#heading=h.u12z2aa14rcl

**Review by Konolo — APPROVED** [review]
LGTM

**Pull request merged**